scripts/todo.js

```diff
@@ -261,46 +263,9 @@ window.onload = function() {
 
         // convert the month names we have installed in the IDB into a month number that JavaScript will understand.
         // The JavaScript date object creates month values as a number between 0 and 11.
-        switch(cursor.value.month) {
-          case "January":
-            var monthNumber = 0;
-            break;
-          case "February":
-            var monthNumber = 1;
-            break;
-          case "March":
-            var monthNumber = 2;
-            break;
-          case "April":
-            var monthNumber = 3;
-            break;
-          case "May":
-            var monthNumber = 4;
-            break;
-          case "June":
-            var monthNumber = 5;
-            break;
-          case "July":
-            var monthNumber = 6;
-            break;
-          case "August":
-            var monthNumber = 7;
-            break;
-          case "September":
-            var monthNumber = 8;
-            break;
-          case "October":
-            var monthNumber = 9;
-            break;
-          case "November":
-            var monthNumber = 10;
-            break;
-          case "December":
-            var monthNumber = 11;
-            break;
-          default:
-          alert('Incorrect month entered in database.');
-        }
+        var monthNumber = MONTHS.indexOf(cursor.value.month);
+        if (monthNumber === -1) alert('Incorrect month entered in database.');
```

Comment:
```
const monthNumber = MONTHS.indexOf(cursor.value.month);
if (monthNumber === -1) throw new Error('Incorrect month entered in database.');
```

Reply:
I agree, but it's too late as I already merged this PR. Sorry I missed that!

Also agree about the terrible `if` statement - but as I said before I really hope there's a better way to implement notifications than this. My inclination would be to remove all the notifications stuff, but we should check that the code isn't being used somewhere in examples.

Reply:
@wbamberg , @tshemsedinov , all `let`s and `var`s are replaced in this [PR](https://github.com/mdn/to-do-notifications/pull/21). There is another PR I am working on which refactors code duplications and refactors some `if` conditions, after that one I could take care of the `Notification` stuff.

Reply:
Thanks! I think we need a prettier here for such a large pieces of code not to miss syntax issues. @valentynpatsera see comments in #21 but please check all this fixes twice because I write all this without running and even without IDE.